exec/plugin-commands-script-runners/test/index.ts

```diff
@@ -437,7 +440,10 @@ test('scripts work with PnP', async () => {
   expect(fooOutput).toStrictEqual(helloWorldJsBinOutput)
 })
 
-test('pnpm run with custom shell', async () => {
+// A .exe file to configure as the scriptShell option would be necessary to test
+// this behavior on Windows. Skip this test for now since compiling a custom
+// .exe would be quite involved and hard to maintain.
+skipOnWindows('pnpm run with custom shell', async () => {
```

Comment:
Note that we're no longer testing custom `script-shell` setups on Windows on CI. This has been broken for the last week due to this issue.

<img width="662" alt="Screenshot 2024-04-16 at 11 13 01 PM" src="https://github.com/pnpm/pnpm/assets/906558/d0051c4f-217e-47c5-99d0-a53d0d129b92">

Reply:
If we do want to support this test, I can compile a `.exe` file to `@pnpm.e2e/shell-mock` that matches the behavior of `shell-mock.js`. Would that be useful, or overkill?

(I actually think we shouldn't do this since it adds hard to audit `.exe` files on CI and locally for pnpm contributors using Windows.)